tests/test_schema_utils.py

```diff
@@ -0,0 +1,39 @@
+from pathlib import Path
+
+import pytest
+
+from tests.schema_utils import (SchemasDirectoryNotFound, get_schema_file_path,
+                                get_schemas_base_path)
+
+
+def test_get_schemas_base_path_returns_base_path(mocker) -> None:
+    expected_path = '/path/to/schemas'
+    mocker.patch('tests.schema_utils.sys.path', [expected_path])
+
+    base_path = get_schemas_base_path(lib_name='schemas')
+    assert base_path == expected_path
+
+
+def test_get_schemas_base_path_raises_exception() -> None:
+    lib_name = 'non-existent'
+    message = f'Unable to find {lib_name} directory. {lib_name} not in sys.path'
+    with pytest.raises(SchemasDirectoryNotFound) as error:
+        get_schemas_base_path(lib_name)
+        assert error.message == message
+
+
+def test_get_schema_file_path_returns_file_path(mocker) -> None:
+    mocker.patch('tests.schema_utils.get_schemas_base_path', new=lambda: '/path/to/schemas/')
+    mocker.patch.object(Path, 'glob', new=lambda self, arg: ['/path/to/schemas/item-list.rng'])
```

Comment:
Not sure if I triggered the change in the tests, but mocking is in the opposite direction of what I suggested, which was integration-testing for this module with the underlying combination of standard library and the Poetry installation of `libero-schemas`. The code smell I see here is a clash with [Only mock types you own](https://markhneedham.com/blog/2009/12/13/tdd-only-mock-types-you-own/) principle, then
:
- happy to explain in person rather than pointing to reference articles
- this doesn't matter much for a 22-line module, but more as a general testing strategy for Python backend services

Reply:
updated.